gateway/api/views/jobs.py

```diff
@@ -112,6 +112,7 @@ def retrieve(self, request, pk=None):  # pylint: disable=unused-argument
         with tracer.start_as_current_span("gateway.job.retrieve", context=ctx):
 
             author = self.request.user
+            return_with_result = request.GET.get("with_result", "true") != "false"
```

Comment:
This is at least the way that we are using to obtain a param from the request:
https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/views/files.py#L68

Reply:
Done!